src/applications/simple-forms/21-4138/config/form.js

```diff
@@ -95,6 +95,7 @@ const formConfig = {
       title: 'What kind of statement do you want to submit?',
       hideFormNavProgress: true,
       hideFormTitle: true,
+      hideOnReviewPage: true,
```

Comment:
can you add `hideOnReviewPage` this to `FormConfigChapter` in our types file? and also `hideFormTitle`?

Reply:
Good catch, fixed! 👍